source/_integrations/tplink.markdown

```diff
@@ -120,10 +124,10 @@ Alternatively, you can factory reset and then prevent the device from accessing
 
 ## Supported functionality
 
-### Cameras
+### Cameras, doorbells and chimes
 
-Only Tapo cameras are currently supported.
-In order for live view to work, you will need to enable your camera account in the Tapo App under **Device Settings** > **Advanced Settings** > **Camera Account**.
+Only Tapo cameras and doorbells are currently supported.
```

Comment:
The section title mentions chimes, but this says only cameras & doorbells are supported. I'm not sure what's the difference between a doorbell and a chime, but we should be consistent on this.